src/lib/server/voice/public/app.js

```
@@ -0,0 +1,268 @@
+    orbEl.className = "listening";
+    statusNode.textContent = "Listening...";
+    toggleBtnEl.textContent = "Stop Conversation";
+    toggleBtnEl.classList.add("active");
+  } catch (err) {
+    // Release any partially-initialized resources
+    if (scriptProcessor) {
+      scriptProcessor.disconnect();
+      scriptProcessor = null;
+    }
+    if (micStream) {
+      micStream.getTracks().forEach((t) => t.stop());
```

Biome (2.4.10)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)